students/N0vA/lesson_04/mailroom2.py

```diff
@@ -11,46 +11,37 @@
             'Howard Schultz': [1000000], 
             'Paul Allen': [450000000]}
 
+# List of donors
+donor_list = list(database.keys())
```

Comment:
this introduced a bug - if you add a new donor and then type "list" command, the new donor is not printed.

this should not be in the global space, just add `print(list(database.keys()))` on line 26